tools/niminst/makefile.nimf

```diff
@@ -124,6 +124,9 @@ ifeq ($(ucpu),powerpc)
     mycpu = $(shell sh -c 'uname -p | tr "[:upper:]" "[:lower:]"')
     CFLAGS += -m64
     LDFLAGS += -m64
+    ifeq ($(mycpu),powerpc64le)
+      mycpu = powerpc64el
```

Comment:
this is redundant as well, why are you comparing to powerpc64le and then assigning mycpu to its exact value? :D